src/webview/LeetCodeWebview.ts

```diff
@@ -30,7 +30,14 @@ export abstract class LeetCodeWebview implements Disposable {
             workspace.onDidChangeConfiguration(this.onDidChangeConfiguration, this, this.listeners);
         } else {
             this.panel.title = title;
-            this.panel.reveal(viewColumn, preserveFocus);
+            if (viewColumn === ViewColumn.Two) {
+                // Make sure second group exists. See vscode#71608 issue
+                commands.executeCommand("workbench.action.focusSecondEditorGroup").then(() => {
```

Comment:
Do we need `await` here?

Reply:
`showWebviewInternal` is not an aysnc function, and the workaround is not needed anymore when the issue mentioned is solved. So I think it not worthful to make it `await`.

Besides, `this.panel.reveal` is just a RPC call and do not wait for the result (investigated in the issue), so it is also not needed to wait for the command.